src/olympia/reviewers/utils.py

```diff
@@ -326,9 +326,31 @@ def get_actions(self, request):
             is_admin_tools_viewer or
             (
                 self.version and
-                not self.version.addon.needs_admin_content_review and
-                not self.version.addon.needs_admin_code_review and
-                not self.version.addon.needs_admin_theme_review
+                (
+                    (
+                        (
+                            # Content Review
+                            self.content_review_only and
+                            (
+                                not self.addon.needs_admin_content_review or
+                                # Content reviews should allowed if the add-on
+                                # is pending admin review because it has
+                                # sources attached
+                                self.version.source
```

Comment:
I don't follow this logic - it's surely possible to have an add-on that is marked `needs_admin_content_review` _and_ have attached sources.